docs/docs/index.md

```diff
@@ -4,31 +4,32 @@ description: The one stop location for tutorials, guides, and information about
 disableTableOfContents: true
 ---
 
-Gatsby is a blazing fast modern site generator for React.
+Gatsby React tabanlı oldukça hızlı bir modern site oluşturucusudur.
 
-## Get Started
+## Başlarken
 
-There are two main ways to get started with Gatsby:
+Gatsby'ye başlamanın 2 yolu var:
```

Comment:
I think it's better to use 2 as `iki` to keep Turkish spelling rules. Also, don't forget the word `main`. Final shape can be something like "Gatsby'ye başlamanın iki ana yolu vardır:". Thanks.